integrations/react/src/StackRefManager.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import type { BaseActivities } from "./BaseActivities";
+import { CoreActionsContext } from "./core/CoreActionsContext";
+import type { StackRefCurrentType } from "./createStackRef";
+import { useActions } from "./useActions";
+
+const StackRefManager = React.forwardRef<
+  StackRefCurrentType<BaseActivities> | undefined,
+  {}
+>((_, ref) => {
+  const actions = useActions();
+  const { dispatchEvent, getStack } = React.useContext(CoreActionsContext);
+
+  React.useImperativeHandle(
+    ref,
+    React.useCallback(
+      () => ({
+        dispatchEvent,
+        getStack,
+        actions,
+      }),
```

Comment:
시그니쳐를 플러그인의 actions 랑 맞추려면 다음과 같이 수정하면 좋을거같아요.

```suggestion
      () => ({
        actions: {
          ...actions,
          dispatchEvent,
          getStack,
        },
      }),
```